### ISSUE TITLE
Conditions in relation to date, time and amount.

### ISSUE THREAD

**Issue by paaljoachim** [CalderaWP/Caldera-Forms#1933]
Conditions on date/time. At a specific date (and time) the form changes.

Use case: Event form, and likely many other form types as well.

Conditions on amount. A certain amount of ticket are for sale. A condition to check for amount of tickets sold in one form and calculate how many tickets have been sold (calculates based on entries etc). At a set amount of tickets reach the predefined limit the form fields are changed to reflect the change.

Use case: Selling tickets, selling products, selling project time, and likely many other form types as well.

Bottom line: conditions for date, time and amount (or something else that is either counted up toward or down to).
